ting_material_details.field.inc

```diff
@@ -457,6 +463,36 @@ function ting_material_details_field_formatter_view($entity_type, $entity, $fiel
             '#markup' => $entity->serieDescription,
           );
           break;
+
+        case 'ting_details_is_part_of_search':
+          $type = $uri = NULL;
+          foreach ($entity->getRelations() as $relation) {
+            $type = explode(':', $relation->getType());
+            if ( isset($type[1]) && $type[1] == 'isPartOfManifestation' ) {
+              $uri  = $relation->getURI();
+            }
```

Comment:
To be honest I don't know. I consider it on the same level as as the rest of the name changing or another relation appearing with the same name but with different semantics.

At least a comment somewhere would be nice. In the current form there is nothing that points to the fact that the type is/can be a namespaced string.